builtin/providers/aws/resource_aws_elb.go

```diff
@@ -371,11 +386,47 @@ func resourceAwsElbUpdate(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	if d.HasChange("idle_timeout") {
+		log.Println("[INFO] inside modify attributes")
+		attrs := elb.ModifyLoadBalancerAttributesInput{
+			LoadBalancerName: aws.String(d.Get("name").(string)),
+			LoadBalancerAttributes: &elb.LoadBalancerAttributes{
+				ConnectionSettings: &elb.ConnectionSettings{
+					IdleTimeout: aws.Integer(d.Get("idle_timeout").(int)),
+				},
+			},
+		}
+		_, err := elbconn.ModifyLoadBalancerAttributes(&attrs)
+		if err != nil {
+			return fmt.Errorf("Failure configuring idle timeout: %s", err)
+		}
+		d.SetPartial("idle_timeout")
+	}
+
+	if d.HasChange("connection_draining") || d.HasChange("connection_draining_timeout") {
+		log.Println("[INFO] inside modify attributes")
+		attrs := elb.ModifyLoadBalancerAttributesInput{
+			LoadBalancerName: aws.String(d.Get("name").(string)),
+			LoadBalancerAttributes: &elb.LoadBalancerAttributes{
+				ConnectionDraining: &elb.ConnectionDraining{
+					Enabled: aws.Boolean(d.Get("connection_draining").(bool)),
+					Timeout: aws.Integer(d.Get("connection_draining_timeout").(int)),
+				},
+			},
+		}
+		_, err := elbconn.ModifyLoadBalancerAttributes(&attrs)
+		if err != nil {
+			return fmt.Errorf("Failure configuring connection draining: %s", err)
+		}
+		d.SetPartial("connection_draining")
+		d.SetPartial("connection_draining_timeout")
+	}
+
```

Comment:
I suppose it depends on how AWS handles this on their side. I know for Route 53 for example you can get errors returned if you make a modification request while another modification request is still in-flight. If ELB are quicker, or don't do that at all, then it's just an optimization _nice to have_. But if for example changing both results in the latter failing, then obviously we need to avoid that. I'm OK leaving as is, assuming that is _not_ the case.